### PR TITLE
add banner support ukraine

### PR DESCRIPTION
**Pull request by juanpicado**
I'm doing all is on my hands to create awareness, seeking for help and spread the voice. This is going to be on the next release v5. Modal opens on click the flag. Flag is not removable and will be there until is necesary. 

First two links open on:
- https://twitter.com/denysdovhan/status/1501486563842211843 (really touching sad podcast)
- https://snyk.io/blog/celebrating-amazing-open-source-innovation-ukraine/

image

### CODE REVIEW

**Review by marekaf — APPROVED** [review]

**Comment by indiescripter** [comment]
I applaud your initiative @juanpicado.

It's difficult to find the right words without upsetting some very nice & innocent Russian OSS developers as well. I think you have found the right balance though striking the humanitarian aspects.

It would be nice, though I don't know the right words, to acknowledge the suffering of innocent Russian people as well. Many war dissenters have been jailed in Russia & may well be sent to Siberia -- we don't know.

Another way people can help is to donate blood to Red Cross. No doubt blood donations are required in those countries, such as Poland, Germany, etc., who are welcoming war displaced refugees from Ukraine. Could we add something about developers donating blood in their respective countries? In Australia this is managed through https://twitter.com/lifebloodau and the Red Cross & I suspect International Red Cross has blood donation centres in just about every western country. Sure you would know what the deal is in Germany for example.

Perhaps others could chime in on my take but of course, no doubt, you are wanting to get this humanitarian message deployed asap.

Once again, good on you for taking this initiative.

**Comment by juanpicado** [comment]
> I applaud your initiative @juanpicado.
> 
> It's difficult to find the right words without upsetting some very nice & innocent Russian OSS developers as well. I think you have found the right balance though striking the humanitarian aspects.
> 
> It would be nice, though I don't know the right words, to acknowledge the suffering of innocent Russian people as well. Many war dissenters have been jailed in Russia & may well be sent to Siberia -- we don't know.

That's political internals affairs and different story, furthermore that's how dictatorships works, I've [lived myself](https://www.hrw.org/world-report/2021/country-chapters/nicaragua) in Nicaragua (from 2006 to 2011).  I don't want to get into politics.

> 
> Another way people can help is to donate blood to Red Cross. No doubt blood donations are required in those countries, such as Poland, Germany, etc., who are welcoming war displaced refugees from Ukraine. Could we add something about developers donating blood in their respective countries? In Australia this is managed through https://twitter.com/lifebloodau and the Red Cross & I suspect International Red Cross has blood donation centres in just about every western country. Sure you would know what the deal is in Germany for example.

> That's a great idea, I'll try to add the right link.

Thanks, just doing the best I can do.

> 
> Perhaps others could chime in on my take but of course, no doubt, you are wanting to get this humanitarian message deployed asap.
> 
> Once again, good on you for taking this initiative.

**Comment by juanpicado** [comment]
Thanks for the feedback, I want to release this today, every minute counts. If after merge this you have more thoughts feel free to drop them here.

**Comment by indiescripter** [comment]
All good & agree with your comments about wanting to stay out of politics. +1 for blood link. Yep, every minute counts. Go for it.

**Review by indiescripter — APPROVED** [review]
So long as it builds & deploys with new blood links LGTM.

File: packages/plugins/ui-theme/src/App/Header/Support/Support.tsx [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+                    {'Learn more about Open Source developers in Ukraine'}
+                  </a>
+                </li>
+                <li>
+                  <a
+                    href="https://www.savethechildren.org/us/where-we-work/ukraine/"
+                    rel="noreferrer"
+                    target="_blank"
+                  >
+                    {'Donate to Save the Childrens'}
```
Comment (DanielRuf) [inline packages/plugins/ui-theme/src/App/Header/Support/Support.tsx:102]:
```suggestion
                    {'Donate to Save the Children'}
```

**Review by DanielRuf — APPROVED** [review]

File: packages/plugins/ui-theme/src/App/Header/Support/Support.tsx (hunk 1 above)
Comment (DanielRuf) [reply packages/plugins/ui-theme/src/App/Header/Support/Support.tsx:102]: Small suggestion.

Comment (juanpicado) [reply packages/plugins/ui-theme/src/App/Header/Support/Support.tsx]: Thanks Daniel 👍🏼

**Comment by northmule** [comment]
I don't welcome your initiative. Developers do not have the competencies and the entire amount of information to make a correct assessment of the events taking place.

**Comment by SergAHell** [comment]
The project is a software product, not a policy. I am very sorry that the developers of the product influenced the thoughts of politicians and joined the political struggle. tomorrow politicians will reconcile and no one will remember that politicians quarreled. But in the program project there will be memory forever that authors can harm their users.

Such actions force others to create a fork in the project and use a code that is safe enough for themselves. Fork essentially suspends receiving new versions from you and your development in general, as the audience becomes smaller.

I personally do not support the course that program projects begin to go into political issues. Politics is a lie anyway.
Please stay professional in your field.

**Comment by EvilBeaver** [comment]
Also, this banner is a half-truth anyway. To be 100% honest and peace-fighting - banner should contain links to Donbass civil war and Madonna of Gorlovka pictures and her history. Then, this banner will be honest and trustworthy. Now it's shows only one side, thus it's biased and not peace-seeking

**Comment by hydra13** [comment]
I disagree with this PR too.

**Pull request merged**